client/dashboard/domains/domain-contact-details/custom-form-fieldsets/get-country-postal-code-support.ts

```diff
@@ -0,0 +1,25 @@
+import { type CountryListItem } from './constants';
+
+/**
+ * Returns true if postal codes are supported on the specified country code.
+ *
+ * NOTE: Will return false if the countries list is empty or if the country
+ * code is not in the list! Always check that the countries list has lo	aded
```

Comment:
```suggestion
 * code is not in the list! Always check that the countries list has loaded
```
I'm guessing the tab is not intended. 🙂

Reply:
Nice eye!